examples/complete/main.tf

```
@@ -0,0 +1,17 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+##############################################################
+# CloudQuery
+##############################################################
+
+module "cloudquery" {
+  source = "../../"
+
+  # Name to use on all resources created (VPC, RDS, etc)
+  name = "cloudquery-complete-example"
+
+  config_file = "${path.cwd}/config.hcl"
```

```suggestion
  config_file = "config.hcl"
```

This is usually enough.